src/components/select/select.scss

```diff
@@ -135,3 +116,13 @@ $mdc-select-bottom-line-hover-color: $lime-text-field-bottom-line-color;
         }
     }
 }
+
+select.mdc-select__native-control {
```

Comment:
Huh? Still something that's both native and mdc?

Reply:
No MDC select should not be used at all, I just reused the same class-name we already had. I can change it if you want

Reply:
It's probably less likely to cause confusion if renamed, yes 😄